test/test_parallel.jl

```diff
@@ -621,9 +621,34 @@
             end
         end
 
-        @test norm(y - A * x)/norm(A * x) < 1e-10
-
+        @test norm(y - A * x) / norm(A * x) < 1e-10
 
     end
 
```

Comment:
let's add a link to this PR in a comment here, so later on people will know why we added this